test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1158,6 +1158,50 @@ ConstructorTestClass(int i, bool b)
         }
     }
 
+    Context "Get-ExperimentalFeature -Name parameter completion" {
+        BeforeAll {
+            function GetExperimentalFeatureNames([switch]$SingleQuote, [switch]$DoubleQuote) {
+                $features = (Get-ExperimentalFeature).Name
+
+                if ($SingleQuote) {
+                    return ($features | ForEach-Object { "'$_'" })
+                }
+                elseif ($DoubleQuote) {
+                    return ($features | ForEach-Object { """$_""" })
+                }
+
+                return $features
+            }
+
+            $allExperimentalFeatures = GetExperimentalFeatureNames
+            $allExperimentalFeaturesSingleQuote = GetExperimentalFeatureNames -SingleQuote
+            $allExperimentalFeaturesDoubleQuote = GetExperimentalFeatureNames -DoubleQuote
+            $experimentalFeaturesStartingWithPS = $allExperimentalFeatures | Where-Object { $_ -like 'PS*'}
```

Comment:
The names are not permanent. I don't want us to have to fix this test every time the list of current features changes. I think we should take the first name and use it for tests, and the test should complete successfully even if the list is empty.

Reply:
Hmm can you give me an example of how this would break? These tests are dynamic so even if features would removed it would still work. If nothing came back for `PS*` features then nothing would come back for completion either.

Reply:
Another option is we see if we can mock `Get-ExperimentalFeature` to always return a hardcoded list.

Reply:
Ok. At least I'd remove `PSNative` - the exp. feature can be removed soon and the test will be dummy.

Reply:
Makes sense. I've removed those tests.